### ISSUE TITLE
Rule proposal: no pending tests

### ISSUE THREAD

**Issue by catkins** [lo1tuma/eslint-plugin-mocha#51]
Inside a large mocha test suite, it's quite difficult to hunt down pending specs of PRs past. It would be awesome to be able to mark them as warnings to pester each other to implement or delete them.

#### OK

```jsx
describe('some module', function() {
  it('implements a test', function() {
    expect(true).to.be.ok;
  });
});
```

### Error

```jsx
describe('some module', function() {
  it('does not yet implement a test');

  it.skip('implemented but skipped', function() {
    expect(false).to.be.ok;
  });
});

describe.skip('another module', function() {
  // ...
});
```

**Comment by lo1tuma** [comment]
Sounds good to me.

**Comment by lo1tuma** [comment]
Implemented in #59 and released with version 2.2.0.
